docs/source/conf.py

```diff
@@ -206,3 +200,5 @@
 smv_tag_whitelist = r'^v.*$'
 smv_branch_whitelist = "master"
 smv_released_pattern = r"refs/tags/v.*"
+
+nbsphinx_execute= 'never'
```

Comment:
will be good to remove/comment this out.

Reply:
🤦 Removed